commands/codeOfConduct.js

```diff
@@ -0,0 +1,39 @@
+let ftl = require('../lib/ftl');
+
+module.exports.register = (botArguments) => {
+  const bot = botArguments.bot;
+  bot.registerCommand('acceptcoc', (msg) => {
+    const adminChannel = process.env.ADMIN_CHANNEL_ID;
+    const cocRole = process.env.COC_ROLE_ID;
+    const userID = msg.member.id;
+    const guildID = msg.channel.guild.id;
+    const reason = 'member accepts the Code of Conduct';
+    const user = `<@${msg.member.id}>`;
+
+    if (adminChannel) {
+      bot.createMessage(adminChannel, { content: ftl('acceptcoc-admin-message', { user }) });
+    }
+    if (cocRole) {
+      bot.addGuildMemberRole(guildID, userID, cocRole, reason);
+    }
+    bot.createMessage(msg.channel.id, { content: ftl('acceptcoc-member-message') });
+  });
+};
+
+//This fires on the member joining event.
+module.exports.onJoin = (botArguments) => {
```

Comment:
Why have this as a separate entry point into the file? You could just as easily put all of this in the `register` function, right?